src/foam/u2/detail/SectionView.js

```diff
@@ -25,11 +25,14 @@ foam.CLASS({
 
   css: `
     .subtitle {
-      color: /*%GREY2%*/ #8e9090';
+      color: /*%GREY2%*/ #8e9090;
```

Comment:
as mentioned in previous comments ... not happy about changing this file in any way. tbh i don't think we need to, despite my emotional attachment to keeping this file as is.

Reply:
I didn't add this property here. I believe this css property is used elsewhere.
@nanopay-arthur might be able be able to address this issue!

Reply:
@nanopay-arthur may be able to address this issue.

Reply:
@nanopay-arthur It would be very appreciated if you give me a heads up if you decide to make changes to 'subtitle' property. I am also using this class to select div element for subtitle in SectionView.